ospo-book/chapters/glossary.md

```diff
@@ -0,0 +1,43 @@
+## Glossary
+
+Welcome to the glossary for the Open Source Program Offices (OSPOs) Book. As you may know, the world of open source is vast and complex, 
+with many different terms and concepts that can be difficult to navigate. Our aim with this glossary is not to provide a definitive or general definition 
+of these terms, but to clarify what we mean specifically under the context of this book.
+
+We believe that it is important to be concise and clear in our definitions, and to reference existing descriptions and resources whenever possible. 
+This will help us avoid creating a glossary that is overly complex or difficult to understand, and will ensure that our readers have access to the best 
+information and resources available.
+
+* **InnerSource:** is the use of open source software development best practices and the establishment of an open source-like culture within organizations
+for the development of its non-open-source and/or proprietary software. https://en.wikipedia.org/wiki/Inner_source
+
+* **InnerSource Principles:** InnerSource Principles are a set of guidelines that provide a framework for organizations 
+to tap into the collective knowledge and expertise of their employees, and to create a culture of collaboration and innovation. 
+Learn more about InnerSource principles: https://innersourcecommons.org/learn/
+
+* **Knowledge Sharing:** Knowledge sharing refers to the exchange of information and expertise between individuals, teams, and organizations. 
+Knowledge sharing is often a key component of open source and innerSource development practices. Learn more about knowledge sharing:https://en.wikipedia.org/wiki/Knowledge_sharing
+
+* **Open Source:** Open source refers to a type of software that allows users to freely use, modify, and distribute 
+the source code of a program that must comply with certain distribution criteria. Learn more about open source: https://opensource.org/osd/
+
+* **Open Source Culture:** Open source culture refers to a set of values and practices that promote collaboration, transparency, 
```

Comment:
From contributor discussion: not mix innersource with open source culture. Maybe changing the existing link to this that mentions the "4 opens" https://openinfra.dev/four-opens/